src/test/kotlin/org/jetbrains/kotlinx/jupyter/test/testUtil.kt

```diff
@@ -31,6 +32,25 @@ import org.jetbrains.kotlinx.jupyter.libraries.parseLibraryDescriptors
 import org.jetbrains.kotlinx.jupyter.log
 import org.jetbrains.kotlinx.jupyter.repl.CompletionResult
 import java.io.File
+import kotlin.collections.Collection
```

Comment:
`kotlin.collections` is a default import and is not needed here, it's interesting how it was added (maybe it's IDEA or Kotlin plugin bug, then it may be reported)

Reply:
Yes, it's a strange thing - I removed it and neither Idea's imports optimisation nor ktlint gave it back